products/views.py

```diff
@@ -216,7 +216,7 @@ def update_category(request, pk):
 @login_required
 def view_categroy(request):
     """ this view is useful to display all categories """
-
+    #
     if not request.user.is_superuser:
```

Comment:
@Yshah112 remove # and follow the naming convention for pagination(line no 226).